### PR TITLE
deprecate(grouping): Remove references to mobile grouping config

### PR DESCRIPTION
**Pull request by armenzg**
We are removing the mobile grouping, so there is no need to refer to it.

### CODE REVIEW

File: static/app/components/events/interfaces/analyzeFrames.spec.tsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ const makeEventWithFrames = (frames: Frame[]): Event => {
     fingerprints: ['58f1f47bea5239ea25397888dc9253d1'],
     groupingConfig: {
       enhancements: 'eJybzDRxY25-UmZOqpWRgZGhroGJroHRBABbUQb_',
-      id: 'mobile:2021-02-12',
+      id: 'newstyle:2023-01-11',
```
Comment (armenzg) [inline static/app/components/events/interfaces/analyzeFrames.spec.tsx:106]: Quick link to the original PR: #47159

File: static/app/components/events/interfaces/performance/anrRootCause.spec.tsx [hunk 2]
```diff
@@ -139,7 +139,7 @@ const makeEventWithThreads = (threads: Thread[]): Event => {
     fingerprints: ['58f1f47bea5239ea25397888dc9253d1'],
     groupingConfig: {
       enhancements: 'eJybzDRxY25-UmZOqpWRgZGhroGJroHRBABbUQb_',
-      id: 'mobile:2021-02-12',
+      id: 'newstyle:2023-01-11',
```
Comment (armenzg) [inline static/app/components/events/interfaces/performance/anrRootCause.spec.tsx:142]: Original PR: #50925

File: static/app/components/events/interfaces/threads.spec.tsx
Comment (armenzg) [inline static/app/components/events/interfaces/threads.spec.tsx:1]: Original PR #29697

**Review by armenzg — COMMENTED** [review]
None of the related PRs make use of the groupingConfig value.

**Review by romtsn — APPROVED** [review]

**Review by priscilawebdev — APPROVED** [review]
code looks good to me

**Review by shruthilayaj — APPROVED** [review]

**Pull request merged**